src/scss/yotpo_stars.scss

```diff
@@ -0,0 +1,8 @@
+@import 'core/functions';
+@import 'core/vars';
+
+@import 'bootstrap/functions';
+@import 'bootstrap/variables';
+@import 'bootstrap/mixins';
```

Comment:
if you are doing this just because you need $primary color for the icon, please just give the color manually, it's not a big a deal but we save a bit for the complation times and complexity